components/profile/hypercerts-tab/hypercerts-tab-content-claimable.tsx

```diff
@@ -1,19 +1,22 @@
-"use client";
-
-import ExploreListSkeleton from "@/components/explore/explore-list-skeleton";
+import { getAllowListRecordsForAddressByClaimedWithMetadata } from "@/allowlists/actions/getAllowListRecordsForAddressByClaimedWithMetadata";
 import { EmptySection } from "@/components/global/sections";
 import UnclaimedHypercertsList from "@/components/profile/unclaimed-hypercerts-list";
-import { useClaimableHypercerts } from "@/hooks/useClaimableHypercerts";
 
-export const ClaimableContent = ({ address }: { address: string }) => {
-  const { data: response, isLoading } = useClaimableHypercerts(address);
+interface ClaimableContentProps {
+  address: string;
+}
 
-  if (isLoading) return <ExploreListSkeleton length={4} />;
-  if (!response) return <EmptySection />;
+export async function ClaimableContent({ address }: ClaimableContentProps) {
+  const response = await getAllowListRecordsForAddressByClaimedWithMetadata({
+    address,
+    claimed: false,
+  });
 
-  const { data } = response;
+  if (!response || !response.data || response.data.length === 0) {
```

Comment:
`!response?.data?.length` is more readable to me, but personal preference.

Reply:
pushed